lib/broccoli/glimmer-app.ts

```diff
@@ -342,6 +347,12 @@ Please run the following to resolve this warning:
     return appTree;
   }
 
+  public getGlimmerEnvironment(): any {
+    let config = this.project.config(this.project.env);
+
+    return config.GlimmerENV || config.EmberENV;
```

Comment:
my thinking was that it's unlikely someone would specify two variables in `config/environment.js` so I decided to fallback.